### DISCUSSION TITLE
Collection seen as entity

### DISCUSSION THREAD

**Discussion by xenoterracide** [mikro-orm/mikro-orm#3725] [Q&A] [answered]
```
    MetadataError: Entity 'Collection' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in StorefrontProductAggregate.variations)
```

```ts
  @ManyToOne()
  readonly variations: Collection<StorefrontProductVariationEntity> = new Collection<StorefrontProductVariationEntity>(
    this,
  );
```

I don't get why this would be an error, if it helps I'm using the mongo driver

**Comment by B4nan** [comment, marked as answer]
`Collection` wrapper is for m:n or 1:m relations, your property is m:1.

**Reply by xenoterracide** [reply]
am I just supposed to use an array then? why would you have a collection for a 1:m? the "one" side would not be a collection, and the many side would also be a many to one.

**Reply by B4nan** [reply]
Have you actually seen the docs? It works the other way around.

https://mikro-orm.io/docs/relationships

**Reply by xenoterracide** [reply]
derp, ugh, I blame all the illness in my head
